src/authenticator.rs

```diff
@@ -324,7 +322,7 @@ where
 
             Command::YKSerial => self.yk_serial(reply),
             Command::YKGetStatus => self.yk_status(reply),
-            Command::YKGetHMAC(req) => self.yk_hmac(req, reply),
+            Command::YKGetHmac(req) => self.yk_hmac(req, reply),
```

Comment:
Not `YkGetHmac`? (note the capitalization of `k`)

Same for `YkSerial` and `YkGetStatus`

Reply:
Ah, good spot